dropbox_rclone/dropbox_rclone/scripts/cs_dropbox_download.py

```diff
@@ -73,6 +80,7 @@ def download(
     dropbox_cs_ver: str,
     data_types: list,
     download_root: Path,
+    dropbox_directory: str = None,
```

Comment:
Just a thought : can dropbox_directory be a Path?  Does it make things more tedious to convert back to str()?

Reply:
Well the dropbox directory variable contains the "dropbox:" part so it's not really a Path. Don't see any reason to convert a string from the API request to a Path to then a string again, simpler to just keep it a string in my opinion.

Reply:
OK, sounds like you thought about it while you were implementing it. Cool.